homework-09/main.js

```diff
@@ -0,0 +1,154 @@
+// 1
+function Human(name, age, gender, nationality, country, listOfCountries) {
+  this.name = name;
+  this.age = age;
+  this.gender = gender;
+  this.nationality = nationality;
+  this.country = country;
+  this.listOfCountries = listOfCountries;
+};
+
+function greeting(greet) {
+  greet = `Hello! My name is ${this.name} I'm ${this.age}, ${this.gender}

```

Comment:
We do not modify or mutate arguments that are incoming in functions. It's a bad coding